src/ert/_c_wrappers/sched/history_source.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class HistorySource(Enum):
+    REFCASE_SIMULATED = 1
+    REFCASE_HISTORY = 2
+    HISTORY_SOURCE_INVALID = 10
```

Comment:
do we still need the int values?

Reply:
I don't think so, just didn't want to change it, but I can :). Also, I think its time to remove the INVALID option

Reply:
Changed now